epaint/src/text/font.rs

```diff
@@ -282,9 +343,34 @@ impl Font {
         let mut cursor_x_in_points = 0.0f32;
         let mut last_glyph_id = None;
 
+        let mut tab_size = super::MAX_TAB_SIZE;
+
         for c in text.chars() {
             if !self.fonts.is_empty() {
-                let (font_index, glyph_info) = self.glyph_info(c);
+                // For tabs it is nice to let them occupy only what's left of the current
```

Comment:
this is very common behavior, but is it "nice"? Does anyone actually like this behavior? It adds a lot of complication for a dubious feature.

If we should do something smart with tabs, we should implement [elastic tabstops](https://nickgravgaard.com/elastic-tabstops/) instead (but not in this PR)

Reply:
I'm still wrapping my head around it, but it looks really interesting